src/main/java/Parser.java

```diff
@@ -0,0 +1,72 @@
+public class Parser {
+
+    private TaskList taskList;
+
+    public Parser(TaskList taskList){
+        this.taskList = taskList;
+    }
+
+    public void ParseCommand(String[] inputs) {
+
+        try {
+            Commands command = GetCommand(inputs[0]);
+            String taskDescription = "";
+            if (inputs.length > 1) {
+                taskDescription = inputs[1];
+            } else if (inputs[0].equals("todo") || inputs[0].equals("deadline") || inputs[0].equals("event")) {
+                throw new DukeException(String.format("The description of %s cannot be empty", command.toString()));
+            }
+
+            switch (command) {
+                case DONE: {
```

Comment:
I think the indentation on this switch statement is not in line with the coding standard